sass/layout/_sidebar-left.scss

```diff
@@ -257,7 +257,7 @@
 
             i {
                 font-size: 16px;
-                color: var(--center-channel-color-48);
+                color: rgba(var(--center-channel-color-rgb), 0.72);
```

Comment:
This is the actual change for https://mattermost.atlassian.net/browse/MM-26869 by going from 0.48 to 0.72 opacity to match the other menus